### PR TITLE
Added BoF and Poster Session info in About page

### PR DESCRIPTION
**Pull request by wavicles**
Please check the info added

### CODE REVIEW

File: layouts/about/single.html [hunk 1]
```diff
@@ -125,7 +125,33 @@ <h1 class="text-white about-pg-head">Want to submit a talk/workshop for the conf
                           class="yellow-text yellow-hovertext"><u>CFP page </u></a> and <a href="https://in.pycon.org/blog/2021/cfp-announcement.html"
                           class="yellow-text yellow-hovertext"><u> blog</u> </a>for more details. </p>
 
+                  
+                
+                <h1 class="text-white about-pg-head">What are BoFs and Poster Sessions? How one can submit these Proposals? </h1>
```
Comment (SaptakS) [inline layouts/about/single.html:130]:
```suggestion
                <h1 class="text-white about-pg-head pt-5">What are BoFs and Poster Sessions? How one can submit these Proposals? </h1>
```

File: layouts/about/single.html [hunk 2]
```diff
@@ -125,7 +125,33 @@ <h1 class="text-white about-pg-head">Want to submit a talk/workshop for the conf
...
+                      Posters are graphical summary of projects or ideas. The poster session provides an opportunity to 
+                    
+                      network with the conference attendees and acts as a starting point for further discussions. This being an online event, 
+                    
+                      poster presentation will be done with a pre-recorded video of 5-10 minutes which will be followed by a short Q & A session.</p>
+                    
+                  <p class="text-grey pt-3">
+
+                      To submit the proposal visit <a href="https://in.pycon.org/cfp/bofs-and-posters-2021/proposals/"
+                          class="yellow-text yellow-hovertext"><u>BoF and Poster Submission Page</u></a> and read our <a href=" "
```
Comment (SaptakS) [inline layouts/about/single.html:152]: Either we need to wait till the blog is merged and published and add the blog link here, or we can just remove the reference to blog in this sentence. @wavicles what do you think?

**Review by SaptakS — CHANGES_REQUESTED** [review]
Looks mostly good to me. Left a few comments

File: layouts/about/single.html (hunk 2 above)
Comment (wavicles) [reply layouts/about/single.html:152]: I guess the blog is ready. Let's add the link and push.

**Pull request merged**